src/web/app/components/preview-session-panel/preview-session-panel.component.html

```diff
@@ -30,7 +30,7 @@
         <ng-container *ngTemplateOutlet="previewInstructorBtn"></ng-container>
       </a>
       <ng-template #previewInstructorBtn>
-        <button id="btn-preview-instructor" class="btn btn-primary" [disabled]="emailOfInstructorToPreview.length === 0 || forDisplayOnly">Preview as Instructor</button>
+        <button style="position:absolute;right:48px" id="btn-preview-instructor" class="btn btn-primary" [disabled]="emailOfInstructorToPreview.length === 0 || forDisplayOnly">Preview as Instructor</button>
```

Comment:
Fix not accepted.
![image](https://user-images.githubusercontent.com/54243224/222737270-f20354ab-c33b-464d-88d9-01166ba36270.png)

Reply:
Just tell me how I can run it locally hence I can fix it properly

Reply:
Our [developer guidelines](https://teammates.github.io/teammates/setting-up.html) contains all the information you need.